### DISCUSSION TITLE
TAC Goals 2024

### DISCUSSION THREAD

**Discussion by tkuhrt** [openwallet-foundation/tac#102] [Ideas]
**What would you like to see the OpenWallet Foundation TAC accomplish in 2024?**

Please provide one response per comment. This will allow others to use the thumbs up feature to vote for the goal to ensure that the TAC focuses on the most critical item first.

Please read the comments before responding. If your idea is the same as someone else's please provide a thumbs up response to that comment. If you would like to add more thoughts, please respond to that comment.

**Comment by tkuhrt** [comment]
List of events for next year -- include in the #events channel on Discord

**Comment by tkuhrt** [comment]
Creating a resource of slides that others have presented at conferences to present OWF

**Reply by tkuhrt** [reply]
Sean to create a drive link
Tracy to create an initial index page on the TAC site.

**Comment by tkuhrt** [comment]
Challenge: helping projects grow within the OWF

**Comment by tkuhrt** [comment]
Challenge: helping projects collaborate and work with one another

**Reply by davidejalexander** [reply]
Is there a difference between different components in the overall architecture and projects that are implementing different components in different languages. Which is higher priority for OWF strategic objectives

**Comment by tkuhrt** [comment]
Engaging with the EUDI and bringing projects into OWF

**Reply by Why-Advisory** [reply]
Transatlantic interoperability very important

**Comment by tkuhrt** [comment]
Interoperability is a topic for us to focus on next year. Not just for the same protocols, but also across protocols.

**Comment by tkuhrt** [comment]
Working closely with other organizations to connect the dots

**Reply by tkuhrt** [reply]
[SIDI](https://sidi-hub.community/)

**Reply by tkuhrt** [reply]
Engaging with Trust Over IP

**Reply by davidejalexander** [reply]
Engaging with MyData Global https://mydata.org/
They are working with European Data Spaces https://dssc.eu/ and looking at wallets and personal data stores as a person centred approach

**Reply by davidejalexander** [reply]
Engaging with OIX https://openidentityexchange.org/members/anon/new.html?destination=%2Findex.html who are also working with SIDIHub

OIX has a global interoperability working group and have mapped seven of the trust frameworks across the Globe for interoperability and they also run a Framework for Wallets working Group to name two key intersection points

**Comment by tkuhrt** [comment]
Publish a white paper documenting our overall architectural considerations for open wallets (started already in the Architecture SIG)

**Reply by talltree** [reply]
This feels like a particularly high priority.

**Reply by tkuhrt** [reply]
https://github.com/openwallet-foundation/architecture-sig/pull/66
